<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1084</width>
    <height>1384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="introPicLabel">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>pic_to_prog.PNG</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="8" column="4">
       <widget class="QLabel" name="labelSlider">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="4">
       <widget class="QLabel" name="aEndlabel">
        <property name="text">
         <string>a, end</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QLabel" name="bEndlabel">
        <property name="text">
         <string>b, end</string>
        </property>
       </widget>
      </item>
      <item row="5" column="7">
       <widget class="QPushButton" name="showParamButton">
        <property name="text">
         <string>Parameter Study</string>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Parameter, step size</string>
        </property>
       </widget>
      </item>
      <item row="5" column="5">
       <widget class="QSpinBox" name="paramSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QSlider" name="ElemSlider">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLineEdit" name="qEdit"/>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="bLabel">
        <property name="text">
         <string>b (m)</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLabel" name="elemLabel">
        <property name="text">
         <string>Maximum Element size</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="eLabel">
        <property name="text">
         <string>E (Pa)</string>
        </property>
       </widget>
      </item>
      <item row="8" column="6">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="8">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="7" column="7">
       <widget class="QPushButton" name="WriteOutPutButton">
        <property name="text">
         <string>Write report to file </string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="7">
       <widget class="QPushButton" name="showGeometryButton">
        <property name="text">
         <string>Geometry</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QLineEdit" name="aEndEdit"/>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="wLabel">
        <property name="text">
         <string>w (m)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="bEdit"/>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="wEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="hLabel">
        <property name="text">
         <string>h (m)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="tEdit"/>
      </item>
      <item row="1" column="7">
       <widget class="QPushButton" name="showMeshButton">
        <property name="text">
         <string>Mesh</string>
        </property>
       </widget>
      </item>
      <item row="2" column="7">
       <widget class="QPushButton" name="showNodalValuesButton">
        <property name="text">
         <string>Nodal Values</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="vLabel">
        <property name="text">
         <string>v (-)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="tLabel">
        <property name="text">
         <string>t (m)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="7">
       <widget class="QPushButton" name="showElementValuesButton">
        <property name="text">
         <string>Element Values</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="aLabel">
        <property name="text">
         <string>a (m)</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="qLabel">
        <property name="text">
         <string>q (N/m)</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLineEdit" name="vEdit"/>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="aEdit"/>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="hEdit"/>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="eEdit"/>
      </item>
      <item row="3" column="5">
       <widget class="QLineEdit" name="bEndEdit"/>
      </item>
      <item row="2" column="6">
       <widget class="QRadioButton" name="aRadioButton">
        <property name="text">
         <string>Vary</string>
        </property>
       </widget>
      </item>
      <item row="3" column="6">
       <widget class="QRadioButton" name="bRadioButton">
        <property name="text">
         <string>Vary</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="reportEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1084</width>
     <height>38</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuCalculate">
    <property name="title">
     <string>Calculate</string>
    </property>
    <addaction name="actionExecute"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCalculate"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionExecute">
   <property name="text">
    <string>Execute</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
